Formula/ccls.rb

```diff
@@ -0,0 +1,37 @@
+class Ccls < Formula
+  desc "C/C++ language server"
+  homepage "https://github.com/MaskRay/ccls"
+  url "https://github.com/MaskRay/ccls.git", :tag => "0.20180513", :revision => "19d0aad2ca9663b5633b575efe41803005b21d38"
+  head "https://github.com/MaskRay/ccls.git"
+
+  option "with-build-debug", "Configures ccls to be built in debug mode"
+  option "without-system-clang", "Downloading Clang from http://releases.llvm.org/ during the configure process"
+
+  depends_on "cmake" => :build
+  depends_on "llvm@6" => :build
```

Comment:
`llvm`, not `llvm@6`. And this is a runtime dependency, not `:build`

Reply:
`ccls` need llvm 6+. What am I supposed to do?

Reply:
llvm is llvm 6 by default.